src/style.css

```
@@ -1,3 +1,107 @@
+html {
+  font-family: Roboto, sans-serif;
+
+  font-optical-sizing: auto;
+
+  font-weight: 500;
+
+  font-style: normal;
+
+  font-variation-settings: 'wdth' 100;
```

```suggestion
  font-family: Roboto, sans-serif;
  font-optical-sizing: auto;
  font-weight: 500;
  font-style: normal;
  font-variation-settings: 'wdth' 100;
```
extra lines are redundant